python/pyspark/sql/tests/test_functions.py

```diff
@@ -60,7 +60,7 @@ def test_function_parity(self):
             "typedLit",  # Scala only
             "monotonicallyIncreasingId",  # depreciated, use monotonically_increasing_id
             "not",  # equivalent to python ~expression
-            "udaf",  # used for creating UDAF's which are not supported in PySpark
+            "udaf", # used for creating UDAF's which are not supported in PySpark
```

Comment:
let's don't touch unrelated files

Reply:
Okay, I see an extra space in front, and I can't help but want to delete it, haha
Let me revert it.

Reply:
Ok, Let me revert it.